mmpose/datasets/pipelines/loading.py

```diff
@@ -71,10 +71,10 @@ def __call__(self, results):
                 # The original results['img'] is assumed to be image(s) in BGR
                 # order, so we convert the color according to the arguments.
                 if results['img'].ndim == 3:
-                    results['img'] = mmcv.bgr2rgb(results['img'])
```

Comment:
Numpy tensor indexing is faster than `mmcv.bgr2rgb` but the cost is 1e-6 sec/image level, which should be negligible. So I think we don't need to change the original code since it's more readable.